tests/x11/gimp.pm

```diff
@@ -18,7 +18,8 @@ use testapi;
 sub run {
     ensure_installed("gimp");
     x11_start_program('gimp');
-    send_key "alt-f4";    # Exit
+    #send_key "alt-f4";    # Exit
```

Comment:
Please, remove commented code and put a comment why we need send_key_until_needlematch.